lib/v2/screens/receive_screens/receive_enter_data/interactor/receive_enter_data_bloc.dart

```diff
@@ -22,21 +26,30 @@ class ReceiveEnterDataBloc extends Bloc<ReceiveEnterDataEvents, ReceiveEnterData
     if (event is OnAmountChange) {
       double parsedQuantity = double.tryParse(event.amountChanged) ?? 0;
 
+      String seedsToFiat =
+          state.ratesState.fromSeedsToFiat(parsedQuantity, settingsStorage.selectedFiatCurrency).fiatFormatted;
+
       if (parsedQuantity > 0) {
-        yield state.copyWith(isNextButtonEnabled: true);
+        yield state.copyWith(isNextButtonEnabled: true, quantity: parsedQuantity, fiatAmount: seedsToFiat);
       } else {
-        yield state.copyWith(isNextButtonEnabled: false);
+        yield state.copyWith(isNextButtonEnabled: false, quantity: parsedQuantity, fiatAmount: seedsToFiat);
       }
     }
     if (event is OnDescriptionChange) {
       yield state.copyWith(description: event.description);
     }
     if (event is OnNextButtonTapped) {
```

Comment:
these should be else if () {} else if (){}

Reply:
fix